doc/user_guide/Geographic_Data.ipynb

```diff
@@ -415,7 +417,7 @@
     "- `global_extent` (default=False): Whether to expand the plot extent to span the whole globe\n",
     "- `project` (default=False): Whether to project the data before plotting (adds initial overhead but avoids projecting data when plot is dynamically updated)\n",
     "- `projection` (default=None): Coordinate reference system of the plot (output projection) specified as a string or integer EPSG code, a CRS or Proj pyproj object, a Cartopy CRS object or class name, a WKT string, or a proj.4 string. Defaults to PlateCarree.\n",
-    "- `tiles` (default=False): Whether to overlay the plot on a tile source. Accept the following values:\n",
+    "- `tiles` (default=False): Whether to overlay the plot on a tile source. If coordinate values fall within lat/lon bounds, auto-projects to EPSG:3857 unless `projection=False`. Accepts the following values:\n",
```

Comment:
Thoughts on removing this cell and only link directly to Customization geographic section to reduce redundancy?

Reply:
I prefer to keep them here for now, I think it's nice to have all the options there within the appropriate context.